js/circuit-breaker-callback.js

```diff
@@ -375,15 +379,19 @@
         editor.addSaveListener(__listenToContentChanges);
     };
 
-    var __transitionToNextImage = function(stepName, imageNum) {
+    var __transitionToNextImage = function(stepName, imageNum, reset) {
```

Comment:
Please add in a comment here on what the 'reset' paramter does and when you would need to set it.  It is not clear from looking at the code in this method.